src/app/client-preview/page.tsx

```diff
@@ -72,6 +73,12 @@ export default async function ClientPreviewPage({
 
   const htmlContent = template({ client })
 
+  const bannerImgUrl = !_settings
+    ? '/images/default_banner.png'
+    : settings?.bannerImage?.url
+
+  console.log(settings?.bannerImage?.url)
```

Comment:
console

Reply:
changes made in the next commit @saugatmaharjan